Sources/Core/Int.swift

```diff
@@ -39,3 +39,10 @@ extension Int: Equatable {
     return Bool(Builtin.cmp_eq_Word(lhs._value, rhs._value))
   }
 }
+
+extension Int: Comparable {
+  @_transparent
+  public static func < (_ lhs: Int, _ rhs: Int) -> Bool {
+    Bool(Builtin.cmp_slt_Word(lhs._value, rhs._value))
```

Comment:
Explicit `return` please (mostly because the rest of the library does so.  It is about style consistency rather than any ideological reason here.  If 5.4+ has the implicit return support, I don't really care which is used as long as it is consistent.  The same applies throughout.

Reply:
I'm using SwiftFormat locally, would it make more sense to add `.swiftformat` to `.gitignore`, or would that config committed to the repo help in any way? I hope I can avoid turning this formatter off completely, maybe some of its settings get me close to the styling standards of uSwift.

Reply:
On a related note, I've set the ruler to 80 columns in my `.vscode/settings.json`, should that config go into `.gitignore`, or would you see any value in storing it in the repository?

Reply:
I don't mind having a `.swiftformat` file thrown into the repository, it shouldn't be particularly large.  The `.vscode` directory ... I suppose would be fine as well, as long as there aren't too many things that it adds in settings wise (I tend to recommend at least one plugin).  Actually, at that point it might be interesting to add a cmake presets file as well to allow VSCode to configure the project properly in release mode.  I wouldn't mind some helpers to allow you to have a set of build rules either.  At that point it should be somewhat similar to what I have in [swift-win32](https://github.com/compnerd/swift-win32).  If you find that you can do a better job than the swift-format that I have there, I'd love to see that applied there as well.  The column ruler I think is a good addition to what I have there though.